wallet/lib/pages/detailed_asset_view/owner_view_view_model.dart

```diff
@@ -1,5 +1,6 @@
 import 'dart:async';
 import 'dart:ui';
+
```

Comment:
Why extra line?

Reply:
dart formatter is adding this extra line.